tests/unit/RelationDAOTest.php

```diff
@@ -1,73 +1,81 @@
 <?php
 
-class RelationDAOTest extends CDbTestCase
+/**
+ * run with:
+ * docker-compose run --rm test ./vendor/codeception/codeception/codecept run unit RelationDAOTest
+ */
+class RelationDAOTest extends \CDbTestCase
 {
     /**
      * @dataProvider relationshipProvider
      */
     public function testItshouldAddReciprocalRelation($relationship, $expected_reciprocal)
```

Comment:
Could capitalise the `s` in `should` in the `testItshouldAddReciprocalRelation` function name.